lib/team_api/api.rb

```diff
@@ -58,5 +59,12 @@ def self.generate_tag_category_endpoints(impl)
     end
 
     private_class_method :generate_tag_category_endpoints
+
+    def self.team_api_schema_location
+      about_yml = Gem::Specification.find_by_name 'about_yml'
+      "#{about_yml.gem_dir}/lib/about_yml/schema.json"
```

Comment:
What happens in the case that the gem isn't found?  Will this method still return a string of some sort or throw an exception?